deploy/hub-of-hubs-spec-transport-bridge.yaml.template

```diff
@@ -73,6 +73,18 @@ spec:
                 secretKeyRef:
                   name: hub-of-hubs-database-transport-bridge-secret
                   key: url
+            - name: TRANSPORT_TYPE
+              value: "$TRANSPORT_TYPE"
+            - name: KAFKA_PRODUCER_ID
+              value: "hub-of-hubs"
+            - name: KAFKA_PRODUCER_HOSTS
```

Comment:
The idea behind the naming was to hint the regular format of : "host1:port1,host2:port2,...". If it doesn't make sense then I'd rather we call it KAFKA_BOOTSTRAP_SERVERS because the name of the configuration is bootstrap.servers.

Reply:
good comment. 
there is actually a difference between `KAFKA_BROKERS` and `KAFKA_BOOTSTRAP_SERVERS`.
let's go with your suggestion - `KAFKA_BOOTSTRAP_SERVERS`. 
putting here the official doc about the difference between these two for documentation, I'm sure you already know this:
https://kafka.apache.org/documentation/#connectconfigs_bootstrap.servers